test/whitePaperModel/mining/borrowMiningTests.spec.ts

```diff
@@ -1188,6 +1195,238 @@ contract("borrowMiningTests", async (accounts) => {
+                            const result = await tokenRegistry.getTokenInfoFromAddress(addressDAI);
+                            const daiTokenIndex = result[0];
+                            await accountsContract.methods["setCollateral(uint8,bool)"](
+                                daiTokenIndex,
+                                true,
+                                {
+                                    from: user2,
+                                }
+                            );
+                            await savingAccount.borrow(addressDAI, borrowAmount, { from: user2 });
+
+                            const user2BalanceAfter = BN(await erc20DAI.balanceOf(user2));
+                            expect(user2BalanceAfter.sub(user2BalanceBefore)).to.be.bignumber.equal(
+                                borrowAmount
+                            );
+                            console.log("------------------ user 2 borrows -------------------");
+                            // Deposit an extra token to create a new rate check point
+                            await savingAccount.fastForward(1000);
+                            await savingAccount.deposit(erc20DAI.address, new BN(10), {
+                                from: user1,
+                            });
+
+                            // 4. Claim the minted tokens
+                            // fastforward
+                            await savingAccount.fastForward(100000);
+                            // Deposit an extra tokens to create a new rate check point
+                            await savingAccount.deposit(erc20DAI.address, new BN(10), {
+                                from: user1,
+                            });
+
+                            const balFINUser2 = await erc20FIN.balanceOf(user2);
+                            console.log("balFINUser2", balFINUser2.toString());
+                            const balFINUser1 = await erc20FIN.balanceOf(user1);
+                            console.log("balFINUser1", balFINUser1.toString());
+
+                            // FIN balance before claim
+                            // User 1
+                            const claimDepositUser1 = new BN(
+                                await savingAccount.claimForToken.call(addressDAI, {
+                                    from: user1,
+                                })
+                            );
+                            console.log("claimDepositUser1", claimDepositUser1.toString());
+
+                            const claimDAIUser1 = new BN(
+                                await savingAccount.claimForToken.call(addressDAI, {
+                                    from: user1,
+                                })
+                            );
+                            console.log("claimDAIUser1", claimDAIUser1.toString());
+
+                            const TotalclaimableAmountUser1 = new BN(
```

Comment:
expect `TotalclaimableAmountUser1`  to be greater than zero.

Reply:
Also ensure that the variable naming format are consistent.

Reply:
Have changed it to camel case `totalClaimableAmountUser1`